core/templates/components/state-editor/state-editor-properties-services/state-editor.service.ts

```diff
@@ -228,7 +233,7 @@ export class StateEditorService {
       interactionId === 'ItemSelectionInput' ||
       interactionId === 'DragAndDropSortInput'
     ) {
-      return (
+      return <AnswerChoice[]>(
         <
           ItemSelectionInputCustomizationArgs|
           DragAndDropSortInputCustomizationArgs
```

Comment:
While we are here can we reformat this (maybe introduce type variable) so that it is more readable.

Reply:
done